src/core/Form/Combobox/Combobox/Combobox.baseStyles.tsx

```diff
@@ -0,0 +1,51 @@
+import { css } from 'styled-components';
+import { defaultThemeTokens as theme } from '../../../theme';
+import { font } from '../../../theme/reset';
+
+export const baseStyles = css`
+  &.fi-combobox {
+    ${font(theme)('bodyText')}
```

Comment:
Does this work if I change e.g. display property of the "top-level" element with style prop? In other words, are the styles wrapped so that display and spacing properties can be safely applied?

Reply:
Can't remember if I have tested that. Need to check this.

What did you find when you tested this?

Reply:
I didn't test this particular case yet. I just know we have had issues with this in the past and didn't see a "style wrapper" anywhere, or at least didn't notice one.

Reply:
I now tested with different values for `display`, having some flex related stuff and using `padding` and `margin` like the following:
```tsx
  <Combobox style={{display: 'flex', flexDirection: 'row', justifyContent: 'space-around', margin: 50, padding: 20}}
...
```
But couldn't notice any breaking of the component or unwanted behaviour.